test/utility/test_util.h

```diff
@@ -58,6 +68,7 @@ typedef struct {
     const char *progname;        /* Truncated program name */
     char *build_dir;             /* Build directory path */
     char *tiered_storage_source; /* Tiered storage source */
+    // char *tiered_storage_ext_config; /* Tiered storage Extension config. */
```

Comment:
If this is no longer needed, remove it rather than comment out.

Reply:
Removed unwanted code